src/approaches/__init__.py

```diff
@@ -1,6 +1,6 @@
 """Default imports for approaches folder."""
```

Comment:
weird, why is this only being caught now (by the linter)?

Reply:
i changed a line in `pylintrc`:
```
# Tells whether we should check for unused import in __init__ files.
- init-import=no
+ init-import=yes
```